README.md

```diff
@@ -314,6 +314,22 @@ edit its security group rules to only allow access to VMs in the
 * `OPENSTACK_AUTH_URL`: Your openstack auth url (required)
 * `OPENSTACK_REGION`: The openstack region to deploy sandboxes in (required)
 
+### AWS S3 Storage
+
+Permissions required for main account are:
+* iam:PutUserPolicy
+* iam:CreateUser
+* iam:CreateAccessKey
+* iam:DeleteUser
+* iam:DeleteAccessKey
+* iam:DeleteUserPolicy
+
```

Comment:
nit: could you put a line in between these two lists so they're separated?  e.g.

Ocim settings are:

Reply:
+1